slick/src/main/scala/slick/jdbc/JdbcType.scala

```diff
@@ -45,3 +49,35 @@ trait JdbcType[@specialized(Byte, Short, Int, Long, Char, Float, Double, Boolean
 
   override def toString = scalaType.toString + "'"
 }
+object JdbcType {
+  object Primitives {
+    // Note implicits for String, BigDecimal and all primitives are already imported when profile.api._ is imported
+    // TODO decide on whether to remove this or not...
+    implicit def booleanJdbcType(implicit types: ApiJdbcTypes): JdbcType[Boolean] = types.booleanJdbcType
```

Comment:
I would like some opinions on whether it makes sense to keep these or not
The reason why this is in `object Primitives` is to demonstrate that these implicits are not needed